platform/ui/tailwind.config.js

```diff
@@ -469,6 +469,29 @@ module.exports = {
       '9/12': '75%',
       '10/12': '83.333333%',
       '11/12': '91.666667%',
+      '1/24': '4.166666667%',
+      '2/24': '8.333333333%',
+      '3/24': '12.5%',
+      '4/24': '16.66666667%',
+      '5/24': '20.83333333%',
+      '6/24': '25%',
+      '7/24': '29.16666667%',
+      '8/24': '33.33333333%',
+      '9/24': '37.5%',
+      '10/24': '41.66666667%',
+      '11/24': '45.83333333%',
+      '12/24': '50%',
+      '13/24': '54.16666667%',
+      '14/24': '58.33333333%',
+      '15/24': '62.5%',
+      '16/24': '66.66666667%',
+      '17/24': '70.83333333%',
+      '18/24': '75%',
+      '19/24': '79.16666667%',
+      '20/24': '83.33333333%',
+      '21/24': '87.5%',
+      '22/24': '91.66666667%',
+      '23/24': '95.83333333%',
```

Comment:
I'm hesitant to create a 24 column grid system just to leverage this for the table columns